compiler/src/dotty/tools/dotc/core/SymDenotations.scala

```diff
@@ -1409,7 +1409,7 @@ object SymDenotations {
 
     /** The symbol overriding this symbol in given subclass `ofclazz`.
      *
-     *  @param ofclazz is a subclass of this symbol's owner
+     *  @param isClass is a subclass of this symbol's owner
```

Comment:
Looks like there's multiple issues here:
- `ofclazz` in the first line of the comment hasn't been renamed
- It should be renamed to `inClass`, not `isClass`
- I don't think this was supposed to be an `@param`, the description looks like it's defining a pre-condition which we normally write using `@pre` comments:
```scala
     *  @pre `inClass` is a subclass of this symbol's owner
```

Reply:
fixed